timestampvm/service.go

```diff
@@ -98,7 +99,8 @@ func (s *Service) GetBlock(_ *http.Request, args *GetBlockArgs, reply *GetBlockR
 	reply.APIBlock.ID = block.ID().String()
 	reply.APIBlock.Timestamp = json.Uint64(block.Timestamp().Unix())
 	reply.APIBlock.ParentID = block.Parent().String()
-	reply.Data, err = formatting.EncodeWithChecksum(formatting.CB58, block.Data[:])
+	data := block.Data()
+	reply.Data, err = formatting.EncodeWithChecksum(formatting.CB58, data[:])
```

Comment:
Why is `reply` of type `*GetBlockReply` rather than `*APIBlock`?

Reply:
it was the old code. changing it.